doc/uftrace-recv.md

```diff
@@ -27,6 +27,37 @@ Data will be sent using `uftrace-record` with -H/\--host option.
 :   Run given (shell) command as soon as receive data.  For example, one can run "uftrace replay" for received data.
 
 
+EXAMPLE
+=======
+The uftrace recv command should be run before sends data by record command.
+
+    # HOST 
+    $ uftrace recv -d recv_data --port 1234
+
+Above command start server with port by given(if you not specify port by `--port` option, it will work at default `8090`.) to receive data from remote client. 
```

Comment:
s/start server/starts a server/

Also please place a space between "given" and parenthesis.

Reply:
Also I think description in the example doesn't need to be verbose - what about changing it just to "(default is 8090)"?  Or maybe it'd be better removing the port from the example for simplicity since most people will just use the default.